code/inc/gfx/element/gfx_mesh.hpp

```diff
@@ -0,0 +1,246 @@
+/**************************************************************************************************/
+/**
+* \addtogroup GFX_ELEMENT
+* @{
+* \details
+* This file provides the public interface for the Mesh Module.
+* 
+* \par COPYRIGHT
+* Copyright (C) 2024 Diego Torres. All rights reserved.
+* This program is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 3 of the License, or
+* (at your option) any later version.
+
+* This program is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+* GNU General Public License for more details.
+
+* You should have received a copy of the GNU General Public License
+* along with this program.  If not, see <http://www.gnu.org/licenses/>.
+**************************************************************************************************/
+
+#pragma once
+
+/*=================================================================================================
+** 1.  REFERENCES
+**===============================================================================================*/
+
+/*=================================================================================================
+** 2.  INCLUDE FILES
+**===============================================================================================*/
+
+#include "gfx/gfx_common.hpp"
+
+#include "gfx/element/gfx_element.hpp"
+#include "gfx/element/gfx_vertex.hpp"
+#include "gfx/shader/gfx_shader.hpp"
+#include "gfx/gfx_glvibuffer.hpp"
+
+/*=================================================================================================
+** 3.  DECLARATIONS
+**
+** 3.1 Macros
+**===============================================================================================*/
+
+/*=================================================================================================
+** 3.2 Types and Classes
+**===============================================================================================*/
+namespace GFX
+{
+    namespace Element
+    {
+        /**************************************************************************************************/
+        /**
+        * \par Details: 
+        */
+        class Mesh : public Element
+        {
+
+        public:
+            /*********************************/
+            // Public type definitions
+            /*********************************/
```

Comment:
Remove this comment.

Reply:
Fixed https://github.com/tdiego1/worldweaver/pull/19/commits/1a0df967e70f1752177ebca99e4618e4ec06c26d